app/src/components/Transfer.tsx

```diff
@@ -146,7 +146,7 @@ export default function Transfer() {
   else if (balanceData?.value === 0n) amountPlaceholder = 'No balance'
   else amountPlaceholder = formatAmount(Number(balanceData?.formatted), 'Longer')
 
-  let receiveAmountPlaceholder = 'Receive Amount'
+  let receiveAmountPlaceholder = 'Receive Token'
```

Comment:
This should be 'amount', not token  👀 

```suggestion
  let receiveAmountPlaceholder = 'Receive amount'
```